cloudtrail-lambda-dynamo-cdk/README.md

```diff
@@ -40,25 +43,23 @@ s.
 ## Testing
 
 Once the CDK stack has deployed successfully, you can take the following steps to ensure the pattern is working appropriately:
-1. Using the AWS CLI, create a bucket using the following command:
+1. Using the AWS CLI, upload the test_file.txt found in the src folder to the S3 bucket of your choosing using the following command:
     ```
-    aws s3api create-bucket --bucket {bucket-name} --region {your AWS region}
+    aws s3 cp test/test_file.txt s3://<bucket-name>
     ```
-    If the S3 bucket creation was successful, you should receive the following response:
+    If the file upload was successful, you should receive the following response:
```

Comment:
if all of these is related to tags, maybe you need to add this to the testing so we can see how the configured keys related to the tags of the resources